zipkin-lens/src/components/DiscoverPage/DiscoverPage.jsx

```diff
@@ -265,39 +264,49 @@ const DiscoverPage = ({ history, location }) => {
   });
 
   return (
-    <React.Fragment>
-      <Box width="100%" display="flex" justifyContent="space-between">
-        <Box display="flex" alignItems="center">
-          <Typography variant="h5">
-            Discover
-          </Typography>
-        </Box>
-        <Box pr={4} display="flex" alignItems="center">
-          <TraceJsonUploader />
-          <TraceIdSearchInput />
+    <>
+      <Box pl={3} pr={3}>
+        <Box width="100%" display="flex" justifyContent="space-between">
+          <Box display="flex" alignItems="center">
+            <Typography variant="h5">
+              Discover
+            </Typography>
+          </Box>
+          <Box pr={3} display="flex" alignItems="center">
+            <TraceJsonUploader />
+            <TraceIdSearchInput />
+          </Box>
         </Box>
+        <GlobalSearch findData={findData} />
       </Box>
-      <GlobalSearch findData={findData} />
-      <Paper className={classes.contentPaper}>
-        <Box overflow="auto" width="100%" height="100%">
-          <AppBar position="static">
-            <Tabs
-              value={tabValue}
-              onChange={handleTabChange}
-              className={classes.tabs}
-            >
-              <Tab label="Traces" className={classes.tab} />
-              <Tab label="Dependencies" className={classes.tab} />
-            </Tabs>
-          </AppBar>
-          { /* 2rem is the height of the Appbar. */}
-          <Box height="calc(100% - 2rem)">
-            {tabValue === tracesTab && <TracesTab />}
-            {tabValue === dependenciesTab && <DependenciesTab />}
+      <Box
+        flex="0 1 100%"
+        marginTop={1.5}
+        marginBottom={1}
+        marginRight={3}
+        marginLeft={3}
+      >
+        <Paper className={classes.contentPaper}>
+          <Box overflow="auto" width="100%" height="100%">
+            <AppBar position="static">
+              <Tabs
+                value={tabValue}
+                onChange={handleTabChange}
+                className={classes.tabs}
+              >
+                <Tab label="Traces" className={classes.tab} />
+                <Tab label="Dependencies" className={classes.tab} />
+              </Tabs>
+            </AppBar>
+            { /* 2rem is the height of the Appbar. */}
```

Comment:
Can we extract a constant for the height of app bar and then we don't even need a comment?